input/profiles/StructureDefinition-publishable-measure-cqfm.json

```diff
@@ -499,7 +499,7 @@
         "path": "Measure.useContext",
```

Comment:
The resolution is to remove the slices for useContext (useContext:qualityProgram) from both the Publishable Measure and Publishable Library profiles.